presto-hive/src/test/java/com/facebook/presto/hive/TestHiveSplitSource.java

```diff
@@ -431,21 +431,17 @@ private TestSplit(int id)
         private TestSplit(int id, OptionalInt bucketNumber)
         {
             super(
-                    "partition-name",
                     "path",
                     0,
                     100,
                     100,
-                    properties("id", String.valueOf(id)),
-                    ImmutableList.of(),
                     ImmutableList.of(new InternalHiveBlock(0, 100, ImmutableList.of())),
                     bucketNumber,
                     bucketNumber,
                     true,
                     false,
-                    ImmutableMap.of(),
-                    Optional.empty(),
-                    false);
+                    false,
+                    new HiveSplitPartitionInfo(properties("id", String.valueOf(id)), ImmutableList.of(), "partition-name", ImmutableMap.of(), Optional.empty()));
```

Comment:
nit: We can store this into a static final field `TEST_PARTITION_INFO` int this class.

Reply:
no because you need the id